template.tpl

```diff
@@ -0,0 +1,2413 @@
+  },
+  "containerContexts": [
+    "WEB"
+  ]
+}
+
+
+___TEMPLATE_PARAMETERS___
+
+[
+  {
+    "type": "SELECT",
+    "name": "ecomAPI",
+    "displayName": "Ecommerce API",
+    "macrosInSelect": false,
+    "selectItems": [
+      {
+        "value": "spEcommerce",
+        "displayValue": "Snowplow Ecommerce"
+      },
+      {
+        "value": "ga4Ecommerce",
+        "displayValue": "Google Analytics 4 Ecommerce"
+      },
+      {
+        "value": "uaEcommerce",
+        "displayValue": "Universal Analytics Enhanced Ecommerce"
+      }
+    ],
+    "simpleValueType": true,
+    "defaultValue": "spEcommerce",
+    "help": "Specify the Snowplow Ecommerce API to use. If you already use Google Analytics 4 ecommerce or Universal Analytics Enhanced Ecommerce to collect information about the shopping behavior of your users, you can use the respective APIs as a way to quickly implement Snowplow Ecommerce without making many changes on your current setup. However to get the full value of it, you should properly plan a full migration towards Snowplow Ecommerce API.",
```

Comment:
I feel this should link to the [transitional API docs](https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/plugins/snowplow-ecommerce/#ga4ua-ecommerce-transitional-api) specifically just to be clear; I think without that it would be easy to think that this can do [Enhanced Ecommerce](https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/plugins/enhanced-ecommerce/) plugin tracking when it very much does not.

Reply:
Maybe rephrase to like:

> Use the native Snowplow Ecommerce API or transitional GA4/UA ecommerce adapter APIs for existing dataLayer implementations using those formats. To get full value from the Snowplow Ecommerce plugin we recommend using the native API when possible.